src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ruleNode.tsx

```diff
@@ -186,6 +197,20 @@ class RuleNode extends React.Component<Props> {
     return getFieldTypes[fieldConfig.type](name, fieldConfig);
   };
 
+  openModal = (formFields: any) => {
+    this.setState({
+      showModal: true,
+      formFields,
+    });
+  };
+
+  closeModal = () => {
+    this.setState({
+      showModal: false,
+      selectedIntegration: null,
```

Comment:
@ceorourke can't we figure out the integration type from the `selectedIntegration` instead of making this fake form field?